### DISCUSSION TITLE
Odd question - another plugin?

### DISCUSSION THREAD

**Discussion by CoyoteRay** [arnoudkooi/SN-Utils#544] [General]
The company I work for will not allow the use of SN Utils because it uses specific commands (Activetab, declarativeContent, storage and executeScript. ...) so I am looking for alternatives.  At a minimum would like ServiceNow to work with VS Code and display the technical names. I would rather have SN-Utils, been using it for years and recommend to all developers 1000% if their employer allows it.

**Comment by arnoudkooi** [comment]
The requested permissions are disclosed in the [PRIVACY.md](https://github.com/arnoudkooi/ServiceNow-Utils/blob/master/PRIVACY.md) file.
I see no alternative, as the permissions all have their purpose.
If there are specific concerns, they can get in touch with me.

Some folks choose to take [this](https://www.reddit.com/r/servicenow/comments/115egg8/comment/lls124k/?context=3) stance ;)

**Reply by CoyoteRay** [reply]
Was afraid that was the answer, I could not find anything thus the reason for the question.  I reviewed the permissions as well and agree 100%.  I did not see a way to accomplish the same thing some other way.  I am just so distraught over this, "Please find a tool to increase developer effectiveness" and I get you cannot ... argh!!!  Thank you for the quick rely.
